fsspec_reference_maker/combine.py

```diff
@@ -274,7 +235,7 @@ def _determine_dims(self):
         self.extra_dims = set(ds.dims) - set(ds0.dims)
         self.concat_dims = set(
             k for k, v in ds.dims.items()
-           if k in ds0.dims and v / ds0.dims[k] == 2
+            if k in ds0.dims and v / ds0.dims[k] == 2
         )
         self.same_dims = set(ds.dims) - self.extra_dims - self.concat_dims
         return ds, ds0, fss
```

Comment:
Is `make_coord()` being preserved in v2, or is it deprecated?

Reply:
The v2 module is a complete rewrite of everything here, and doesn't depend on anything that went before. Clearly I didn't leave the PR in a good state to have mixed changed in combine and combine_v2 - but you can scavenge anything that might be useful.

Reply:
OK, thanks. Was the plan to replace `combine.py` with `combine_v2.py`, or supplement it? Since `make_coord()` was defined public, I didn't want to remove it in the event it was considered part of the kerchunk API.

Reply:
As much as there was a plan, the new code was supposed to replace the previous version, so long as it could handle all the existing use cases. It is fine to change the API, and this method in particular would not have been called by users anyway. Yes, it would be a good idea at this point to define what we do expect the users to call exactly. But the whole repo is experimental and not used by many people so far, so we can make as many breaking changes as we need.

Reply:
Excellent. Thanks for your help.